### PR TITLE
FEAT: add YAML release to MyGet

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to #47

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Would be good to configure the build in Azure DevOps so that it runs already here as well.

File: build/myget-release.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    value: 'Arcus.WebApi'
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
```
Comment (tomkerkhove) [inline build/myget-release.yml]: You should determine the package version for the PR

File: build/myget-release.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Compile'
+            inputs:
+              projects: 'src/*.sln'
+              arguments: '--configuration $(Build.Configuration)'
```
Comment (tomkerkhove) [inline build/myget-release.yml]: You are not stamping it with the correct suffix

```suggestion
              arguments: '--configuration $(Build.Configuration) --version-suffix $(packageVersion)'
```

**Comment by stijnmoreels** [comment]
> Looks good to me, do you have this running somewhere because I don't see it in the Arcus Azure DevOps project?

Ah, I removed the build definition after I got it running.

**Comment by tomkerkhove** [comment]
Feel free to reconfigure it given we'll need it once it's merged.

**Pull request merged**